atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/CachingTransaction.java

```diff
@@ -114,7 +118,11 @@ public Transaction delegate() {
     @Override
     public Map<Cell, byte[]> get(TableReference tableRef, Set<Cell> cells) {
         if (cells.isEmpty()) {
-            log.info("Attempted get on '{}' table with empty cells argument", tableRef);
+            if (log.isTraceEnabled()) {
+                log.trace(String.format("Attempted get on %s table with empty cells argument", tableRef), new Exception());
+            } else if (log.isDebugEnabled()) {
+                log.debug("Attempted get on '{}' table with empty cells argument", tableRef);
```

Comment:
I'd prefer to leave this one at info level so that we can find future instances where we call these methods unnecessarily.

Minor preference to always log this one, regardless of whether or not trace is enabled, it's a bit funky that enabling trace will disable info logging.